docs/libvirt-pod-networking.md

```diff
@@ -4,7 +4,7 @@ Author: Fabian Deutsch \<fabiand@redhat.com\>
 
 ## Introduction
```

Comment:
Kubernetes has multiple solutions for pod connectivity and ipam. Inventing a completely new way for connecting VMs is not out of the question, but the simplest most immediate solution is to integrate the VM with the existing solution on the cluster.

The VM has to know and use the pod IP because many CNI plugins provide only L3 connectivity. If the VM uses any other IP address, its traffic would not be routed back and forth.

Since that was not clear from the suggested text, Fabian should explain this better.

Multiple addresses per pod is in the making https://github.com/kubernetes/kubernetes/issues/27398